docs/commerce/checkout/customize_checkout.md

```diff
@@ -0,0 +1,202 @@
+Shut down the application, clear browser cache, and restart the application.
+You should be able to see a different checkout applied after you have added products to a cart.
+
+## Create a single-form checkout
+
+Another way of customizing the process would be to implement a single-form checkout.
+Such solution could work for certain industries, where simplicity is key.
+The single form's basic advantage is simplified navigation with less clicks to complete the transaction.
+
+### Define workflow
+
+To create a single-form checkout, define a workflow that has two stages, `initialized` and `completed`, and one transition, from `initialized` or `completed` to `completed`.
+
+``` yaml hl_lines="3 18 19"
+[[= include_file('code_samples/front/shop/checkout/config/packages/ibexa.yaml', 25, 27) =]] [[= include_file('code_samples/front/shop/checkout/config/packages/ibexa.yaml', 84, 103) =]] [[= include_file('code_samples/front/shop/checkout/config/packages/ibexa.yaml', 23, 24) =]]
+```
+
+### Create controller
+
+Add a controller in project code that is a regular Symfony controller, which reuses classes provided by the application.
+Within the controller, create a form that contains all the necessary fields, such as the shipping and billing addresses, as well as shipping and billing methods.
+
+In the `src/Controller/Checkout` folder, create a file that resembles the following example:
+
+``` php
+[[= include_file('code_samples/front/shop/checkout/src/Controller/SinglePageCheckout.php') =]]
+```
+
+Again, after you handle the form and validate it, you advance the workflow by using the `advance` method provided in the base component.
+
+### Create Twig template
+
+Create a Twig template to render the Symfony form.
+In `templates/themes/custom/storefront/checkout`, create a layout that iterates through all the fields and renders them.
+
+```html+twig
+[[= include_file('code_samples/front/shop/checkout/templates/themes/storefront/checkout/checkout.html.twig') =]]
+```
+### Configure repository
+
+Then you have to map the single-step workflow to the repository, 
+by replacing the default `ibexa_checkout` reference with one of `single_page_checkout`:
+
+``` yaml
+ibexa:
+    repositories:
+        default: 
+            checkout:
+                workflow: single_page_checkout
+```
+
+### Restart application
+
+To see the results of your work, shut down the application, clear browser cache, and restart the application.
+You should be able to see a single-page checkout applied after you add products to a cart.
+
+## Define custom Address Field Type formats 
+
+To create custom Address Field Type formats to be used in checkout, make the following changes in the project configuration files. 
+
+First, define custom format configuration keys for `billing_address_format` and `shipping_address_format`:
+
+``` yaml 
+ibexa:
+    repositories:
+        <repository_name>:
+            checkout:
+                #coming from Corporate Account, "billing" by default
```

Comment:
Can you avoid mentioning corporate account here? It's confusing if the reader doesn't use it.